kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,165 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
```

Comment:
Didn't somebody else contribute to this?

Reply:
well, I took some inspirations yet I acknowledged the author. I think mentioning him here would be confusing